src/main/java/no/ks/fiks/io/client/SvarSender.java

```diff
@@ -39,22 +41,49 @@ public SendtMelding svar(String meldingType, List<Payload> payloads) {
                                         .build(), payloads.isEmpty() ? Option.none() : Option.of(encrypt.apply(payloads))));
     }
 
+    public SendtMelding svar(String meldingType, List<Payload> payloads, MeldingId klientMeldingId) {
+        return SendtMelding.fromSendResponse(utsendingKlient.send(
+            MeldingSpesifikasjonApiModel.builder()
+                .avsenderKontoId(meldingSomSkalKvitteres.getMottakerKontoId().getUuid())
+                .mottakerKontoId(meldingSomSkalKvitteres.getAvsenderKontoId().getUuid())
+                .svarPaMelding(meldingSomSkalKvitteres.getMeldingId().getUuid())
+                .meldingType(meldingType)
+                .headere(ImmutableMap.of(Melding.HeaderKlientMeldingId, klientMeldingId.toString()))
+                .build(), payloads.isEmpty() ? Option.none() : Option.of(encrypt.apply(payloads))));
```

Comment:
Var kanskje litt mye duplikat-kode med en del logikk for disse to send-med-payloads-metodene, hvor bare .headere(..) er forskjell.
Mulig å bygge opp utsendingKlient.send(..) uten like mye duplikat-kode?

Reply:
Tar du en ny titt?

Reply:
Bra, ble enklere å se forskjell på de